packages/material-ui-utils/src/capitalize.ts

```diff
@@ -0,0 +1,13 @@
+// TODO: error TS7016: Could not find a declaration file for module '../macros/MuiError.macro'. '/tmp/material-ui/packages/material-ui-utils/macros/MuiError.macro.js' implicitly has an 'any' type.
+// import MuiError from '../macros/MuiError.macro';
```

Comment:
@eps1lon FYI we have this issue with the macros

Reply:
Could you `@ts-expect-error` this instead for now so that we don't sacrifice the runtime behavior for typings?